server/images.go

```diff
@@ -485,9 +485,18 @@ func CreateModel(ctx context.Context, name, modelFileDir string, commands []pars
 					if err != nil {
 						return err
 					}
-					if err := PullModel(ctx, parent.OriginalModel, &RegistryOptions{}, fn); err != nil {
-						log.Printf("error pulling model: %v", err)
+
+					switch {
```

Comment:
I think this would be slightly easier to read w/o the full switch statement. Something like:

```
originalModel := parent.OriginalModel
if originalModel == "" {
    originalModel = parent.ShortName
}

if err := PullModel(ctx, originalModel, ...)
```